packages/widgets/src/weather/icon.tsx

```diff
@@ -42,13 +43,21 @@ interface WeatherDescriptionProps {
 
 /**
  * Description Dropdown for a given set of parameters
+ * @param dateFormat date format from options
```

Comment:
`apple is apple` -> not very helpful description of an apple, no?
I recommend you to choose a more detailed description like ``format of the date that will be displayed on the widget``